docs/user/algo_bc.md

```diff
@@ -0,0 +1,66 @@
+# Behavioral Cloning
+
+```eval_rst
+.. list-table::
+   :header-rows: 0
+   :stub-columns: 1
+   :widths: auto
+
+   * - **Paper**
+     - Model-Free Imitation Learning with Policy Optimization :cite:`ho2016model`
+   * - **Framework(s)**
+     - .. figure:: ./images/pytorch.png
+        :scale: 10%
+
+
+        PyTorch
+   * - **API Reference**
+     - `garage.torch.algos.BC <../_autoapi/garage/torch/algos/index.html#garage.torch.algos.BC>`_
+   * - **Code**
+     - `garage/torch/algos/bc.py <../_modules/garage/torch/algos/bc.html#BC>`_
+   * - **Examples**
+     - :ref:`bc_point`, :ref:`bc_point_deterministic_policy`
+
+```
+
+Behavioral cloning is a simple immitation learning algorithm which maxmizes the likelhood of an expert demonstration's actions under the apprentice policy using direct policy optimization. Garage's implementation may use either a policy or dataset as the expert.
+
+
+## Default Parameters
+
+```python
+policy_optimizer = torch.optim.Adam
+policy_lr = 1e-3
+loss = 'log_prob'
+batch_size = 1000
+```
+
+## Examples
+
+### bc_point
+
+```eval_rst
+.. literalinclude:: ../../examples/torch/bc_point.py
+```
+
+### bc_point_deterministic_policy
+
+#### Experiment Results
+
+<!-- Copy the image and paste it on github issues to get the github user content link -->
+<img src="https://user-images.githubusercontent.com/15152847/87868865-60b07d80-c94f-11ea-801f-a7ebaf566426.png" width=45%> <img src="https://user-images.githubusercontent.com/15152847/87868876-7a51c500-c94f-11ea-8700-3b0a6142aa96.png" width=45%>
```

Comment:
can you commit these to the repo instead? We have no guarantee these won't disappear or change.